@types/frame/rpc.d.ts

```diff
@@ -80,6 +80,45 @@ declare namespace RPC {
     }
   }
 
+  namespace GetEthereumChains {
+    interface Icon {
+      url: string
+      width?: number
+      height?: number
+      format?: 'png' | 'jpg' | 'svg'
+    }
+
+    interface NativeCurrency {
+      name: string
+      symbol: string
+      decimals: number
+    }
+
+    interface Explorer {
+      name?: string
+      icon?: Icon[]
```

Comment:
`icon` meant for `NativeCurrency`?

Reply:
nope, this is the spec. I'd guess the icon of the currency is always assumed to be that of the chain itself.

https://github.com/ethereum-lists/chains#example